kong-2.8.0-0.rockspec

```diff
@@ -72,6 +72,7 @@ build = {
     ["kong.clustering.control_plane"] = "kong/clustering/control_plane.lua",
     ["kong.clustering.wrpc_data_plane"] = "kong/clustering/wrpc_data_plane.lua",
     ["kong.clustering.wrpc_control_plane"] = "kong/clustering/wrpc_control_plane.lua",
+    ["kong.clustering.wrpc_check"] = "kong/clustering/wrpc_check.lua",
```

Comment:
I wonder if it is necessary to add a new file for such a simple function, can it just be inside `wrpc_data_plane.lua`?

Reply:
good idea, I move it to init.lua

Reply:
good idea, I move it to init.lua